Documentation/NetworkBoot.md

```diff
@@ -165,7 +165,7 @@ Add the following file in the root folder of the project:
 #!ipxe
 console --x 1280 --y 1024
 dhcp
-kernel http://X.Y.Z.W/Kernel serial_debug root=/dev/ramdisk0
```

Comment:
I would leave the exposed name the same, the user should not be concerned how our naming convention for static variables is

Reply:
Oh, did it automatically change that *again*? I thought I specifically edited that out of the final one.